### DISCUSSION TITLE
Welcome to Plateforme-e-commerce Discussions!

### DISCUSSION THREAD

**Discussion by nmissi-nadia** [nmissi-nadia/Plateforme-e-commerce#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MohamedMoustir** [comment]
hi

**Comment by Abderrahman-Ahouari** [comment]
i like this

**Comment by nmissi-nadia** [comment]
vos états d'avancement SVP.

**Reply by Dadssi** [reply]
en ce qui concerne la page panier, je suis presque 60% de la version mobile

**Reply by MohamedMoustir** [reply]
j'ai termin header

**Comment by oussamaamou** [comment]
j'ai commence par intégrer la maquette du version Mobile (j'ai presque termine cette partie) , après je vais commencer la version Tablette puis Ordinateur.
